tests/migration_test.go

```diff
@@ -804,14 +855,14 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				By("Checking that the migrated VirtualMachineInstance console has expected output")
 				expecter, err = tests.ReLoggedInFedoraExpecter(vmi, 60)
 				defer expecter.Close()
-				Expect(err).ToNot(HaveOccurred())
+				Expect(err).ToNot(HaveOccurred(), "Should be able to re-login to the migrated VM")
```

Comment:
I think "Should stay logged in after migration" makes more sense

Reply:
Done